### PR TITLE
Signup user: Create new account 

### PR DESCRIPTION
**Pull request by TanBinh1601**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/???)

## WHAT (optional)
- Add sign up function

## HOW
- Add User Controller
- Add error message

## Evidence (Screenshot or Video)
Screenshot from 2021-11-22 00-21-54
Screenshot from 2021-11-22 00-14-32
Screenshot from 2021-11-22 00-20-03

### CODE REVIEW

**Comment by TanBinh1601** [comment]
ready

**Comment by TanBinh1601** [comment]
ready

File: app/helpers/users_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```
Comment (quoctd-0607) [inline app/helpers/users_helper.rb]: không dùng thì bỏ đi e

Comment (TanBinh1601) [reply app/helpers/users_helper.rb]: Dạ

**Comment by TanBinh1601** [comment]
ready

**Pull request merged**